SR2023/2022-10-18-discord-credentials.md

```diff
@@ -0,0 +1,16 @@
+---
+to: Student Robotics 2023 teams
+subject: Student Robotics Discord Forum Details
+---
+
+Hi
+
+This year, we're using [Discord](https://studentrobotics.org/docs/team_admin/discord) as our primary communication method for competitor support. Your invite link is [discord link](). If you don't already have a Discord account, you can register an account for free.
+
+Upon joining the server users will need to provide a password to gain the correct permissions. The password is used to identify which team a user is from, so do not share to those outside your team. For your team, the password is: `password`. Please share both the invite link and the relevant password with your competitors.
+
+More information on this can be found in [the docs](https://studentrobotics.org/docs/team_admin/discord).
+
+Additionally, we like to make sure we are aware which members are volunteers, competitors and team supervisors. To get the 'Team Supervisor' role on Discord please reply to this email with your Discord username once you have joined the server and we will add the appropriate roles.
+
+Whilst most of the discussions and support will be happening on Discord, if you are unable to or do not wish to create an account with Discord you can still [contact us](mailto:teams@studentrobotics.org) for support, however the response times will be longer.
```

Comment:
While not announced yet, the competition this year will involve things that they need to post in Discord to achieve. Perhaps add something like:
> Additionally for SR2023 some elements of the competition will require posting to Discord, so please encourage your team to join. If you are unable to do this, please [let us know](mailto:teams@studentrobotics.org) so we can work out alternative ways to accommodate your team.

Reply:
https://github.com/srobo/team-emails/pull/116/commits/5bcab6432437c260aa3e84c8bc71efc77cb57197